centralbank.py

```diff
@@ -13,6 +14,26 @@ def __init__(self):
         self.twelvemonth_CPI = 0
         self.feedback_counter = 0
         self.prices_list = []
+        self.economy_money = money_supply
+        self.warnings = {}
+        self.aid_budget = 1000000
+
+    def update_money_supply(self, amount, reduce=True):
```

Comment:
Method might need a description. But I would argue that the economy's money should be in the `InsuranceSimulation` object, not in the central bank anyway. See my comment above.